src/Interpreters/Cache/QueryCache.h

```diff
@@ -24,6 +24,15 @@ bool astContainsNonDeterministicFunctions(ASTPtr ast, ContextPtr context);
 class QueryCache
 {
 public:
+    enum class Usage
+    {
+        /// starts at 1 for compatibitity with DataTypeEnum8
```

Comment:
I guess I was confused because DataTypeEnum8 starts with 1 by default.

I'll now
- change to 0 in C++ (the most natural start value), and
- omit the integer constant (0 or 1) from the SQL docs (user don't need to care about the integer constants) - the starting value will now be 0.